config.py

```diff
@@ -33,6 +34,10 @@ def __init__(self):
         self.create_xray_when_sending.setChecked(prefs['create_xray_when_sending'])
         self.l.addWidget(self.create_xray_when_sending)
 
+        self.expand_aliases = QCheckBox('Auto associate split aliases')
```

Comment:
This setting only affects aliases, right?  ie. we have an implicit "auto-expand characters" tick box that is always selected, it is only "auto-expand aliases" that we are letting the user control.

Naming is always a pain when you are trying to be short :)

I discovered today QWhatsThis - this can be another option for avoiding ambiguity, in a way that is linked to the code.  It is dead simple to add, and lets you write a little more: http://doc.qt.io/qt-5/QWhatsThis.html